spec/integration/data_stream/manifest.spec.yml

```diff
@@ -307,6 +332,29 @@ spec:
         dynamic_namespace:
           description: When set to true, agents running this integration are granted data stream privileges for all namespaces of its type
           type: boolean
+    routing_rules:  # technical preview
+      description: Routing rules set.
+      type: array
+      items:
+        type: object
+        additionalProperties: false
+        properties:
+          dataset:
```

Comment:
The option in the processor is called `dataset`, right? Though it seems it could accept a package name.
I would use `dataset`, to be coherent with the processor, or `name` if we want to be generic enough, but maybe `name` is too generic (would it be the option passed to the processor or would it be a name for the route itself?).

Reply:
If I understand correctly the docs and the example in the issue https://github.com/elastic/package-spec/issues/514, this field would define the source dataset to be used by the reroute processor to check for documents to be routed to another dataset. This latter dataset would be the ones defined in each element under `rules` key.

Being that, would `dataset` be a good name? another options that I can think of `source`, `source_dataset`? @jsoriano @juliaElastic 

If so, I would add these description to these fields
```diff
--- spec/integration/data_stream/manifest.spec.yml
+++ spec/integration/data_stream/manifest.spec.yml
@@ -364,8 +364,10 @@ spec:
         additionalProperties: false
         properties:
           dataset:
+            description: Source dataset to be used for this reroute processsor
             type: string
           rules:
+            description: List of routing rules
             type: array
             items:
               $ref: "#/definitions/routing_rule"
```

Reply:
I think `source_dataset` is the clearest name, I vote for that.